Organization/Organization/CreateEmployeeViewController.m

```diff
@@ -0,0 +1,29 @@
+#import "CreateEmployeeViewController.h"
+#import "Employee.h"
+
+
+@interface CreateEmployeeViewController ()
+@property (weak, nonatomic) IBOutlet UITextField *firstNameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *lastNameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *salaryTextField;
+
+@end
+
+@implementation CreateEmployeeViewController
+
+
+- (IBAction)saveEmployee:(id)sender
+{
+    NSString *firstName = _firstNameTextField.text;
```

Comment:
Access properties through getters (`self.`), instead of accessing the variable directly.